README.md

```diff
@@ -67,3 +67,17 @@ public pull request.
 
 Pulls the upstream repository into the local repository.
 
+## [`openlane_run`](./openlane_run)
+
+The goal of this action is to run a given design through the OpenLane flow
+using a given tag. Optionally a PR can be created to update a stable tag
+file stored in the repository.
+Arguments:
+-   `ol_tag` [optional, default: `latest`]: OpenLane tag. You can set to
+    `ol_tag_file` to use the value from the input.
+-   `update_tag` [optional, default: `false`]: if the test is successful
```

Comment:
The intent is to have interdependent workflows as you said:
- Test any changes made to the design with the last known good version (i.e., `ol_tag_file`).
- Test each new release of OpenLane with the user's design, if it passes give the choice to update the latest tag to be the stable tag (i.e., master/HEAD -> `ol_tag_file`).

I would be ok with removing `ol_tag` and `update_tag` in favour of a `mode` with the following possible values: `stable` (uses the file, does not update the file), `latest` (uses master/HEAD, tries to update file). But this will reduce the flexibility of the action. I can think of at least one use case that would not be possible, e.g., users cannot create a dispatch workflow to test with arbitrary tags/patches. Note that I am not the intended demographic, so this is just my best effort to support them.